src/cls/_ZPM/PackageManager/Developer/Document/Module.cls

```diff
@@ -7,6 +7,10 @@ Projection RegisterExtension As %ZPM.PackageManager.Developer.Document.Projectio
 
 Parameter EXTENSION = ".MODULE";
 
+// As long as we are still supporting both .MODULE & .ZPM extension, we need a default extension to use when exporting/creating. Change it according to your use case.
+
+Parameter DEFAULTEXTENSION = ".ZPM";
```

Comment:
So does this mean we will use ZPM wherever possible instead of module? I am beginning to wonder if it may just be better to change all the module.xml files for HS to have the ZPM extension